LEAF_Request_Portal/admin/Group.php

```diff
@@ -88,14 +88,21 @@ public function getMembers($groupID)
             $dir = new VAMC_Directory();
             foreach ($res as $member)
             {
-                $dirRes = $dir->lookupLogin($member['userID']);
+                $dirRes = $dir->lookupLogin($member['userID'], false, true);
 
                 if (isset($dirRes[0]))
                 {
+                    $dirRes[0]['regionallyManaged'] = false;
                     if($groupID == 1)
                     {
                       $dirRes[0]['primary_admin'] = $member['primary_admin'];  
                     }
+                    foreach ($dirRes[0]['groups'] as $group)
+                    {
+                        if ($groupID == $group['groupID']) {
```

Comment:
This could also be:
`$dirRes[0]['regionallyManaged'] = $groupID == $group['groupID']`

Reply:
Makes sense to me. Changing it now.